sherpa/stats/tests/test_stats.py

```diff
@@ -299,6 +299,15 @@ def test_wstat_error(self):
         fit = Fit(data, self.model, WStat(), NelderMead())
         self.assertRaises(StatErr, fit.fit)
 
+    def test_chi2datavar(self):
+        num = 3
+        xy = numpy.array(range(num))
+        ui.load_arrays(1, xy, xy, Data1D)
+        ui.set_stat('chi2datavar')
+        err = ui.get_staterror()
+        for index, yyy in enumerate(xy):
```

Comment:
I should also note that we do have Sherpa specific "test an array for equality" routines  somewhere in `SherpaTest or `SherpaTestCase` but I think these should probably be marked as deprecated, since there seems little advantage to them compared to the numpy ones (but this needs review, and is not really part of this particular PR).